wom/src/main/scala/wom/graph/expression/AnonymousExpressionNode.scala

```diff
@@ -23,4 +26,8 @@ object AnonymousExpressionNode {
 /**
   * An expression node that is purely an internal expression and shouldn't be visible outside the graph
   */
-trait AnonymousExpressionNode extends ExpressionNode
+trait AnonymousExpressionNode extends ExpressionNode {
+  val callNodeReceivingInput: GraphNodeSetter[CallNode]
+
+  def feedsTaskCallNode: Boolean = callNodeReceivingInput.isAssignedAndOfType[TaskCallNode]
```

Comment:
Oh, I see, the field isn’t set for Ifs and Scatters?

I’m a little uneasy about “how do we know it wasn’t a task but just wasn’t set yet”... but I think it’s pretty safe. Hopefully!

Reply:
I think that should be safe since those relationships are established when the task call is built, but the WEA doesn't look at this until after the task call is built and in the execution store.